src/App.css

```
@@ -0,0 +1,16 @@
+body {
+  background: #1b1c1d;
+}
+
+.isStarted-button {
```

```suggestion
.started-button {
```
классы стилей пишем в нижнем регистре